example/dummy/persistent_dummy.go

```diff
@@ -38,14 +42,19 @@ func NewPersistentDummyApplication(dbDir string) *PersistentDummyApplication {
 	stateTree := iavl.NewIAVLTree(0, db)
 	stateTree.Load(lastBlock.AppHash)
 
-	log.Notice("Loaded state", "block", lastBlock.Height, "root", stateTree.Hash())
+	// log.Notice("Loaded state", "block", lastBlock.Height, "root", stateTree.Hash())
```

Comment:
Actually thats not a bad idea. Would avoid the ugly SetLogger with the type assertion too